### DISCUSSION TITLE
Expression of Interest: Enhancing Generative AI Workloads on ARM Devices with OpenVINO [Project Idea Number: 8]

### DISCUSSION THREAD

**Discussion by AtharvaPore01** [openvinotoolkit/openvino#22714] [Google Summer of Code]
I am writing to express my keen interest in contributing to the project aimed at improving Generative AI (GenAI) workload performance on ARM devices with OpenVINO. As an enthusiast in the field of artificial intelligence and a seasoned C++ developer with over five years of experience, I find this project particularly intriguing. With my background in low-level programming and proficiency in languages like C++ and Python, I am equipped with the necessary skills to tackle the challenges posed by this project effectively. Additionally, I currently use a MacBook Pro with an M1 chip, providing me with a suitable environment for development and testing.
Moreover, I have already started contributing to the OpenVINO project by addressing ISSUE: 22313 (https://github.com/openvinotoolkit/openvino/issues/22313), which is presently under review. This experience has acquainted me with the project's codebase and its development workflow, making me well-prepared to take on more substantial tasks within the OpenVINO ecosystem.
I am excited about the opportunity to work under the mentorship of Alexandr Voron and Dmitry Gorokhov and to collaborate with the OpenVINO community to optimize GenAI workloads for ARM devices. I believe that my skills, combined with the guidance of experienced mentors, will enable me to make meaningful contributions to the project and achieve the desired outcomes, including better compilation and runtime performance, as well as reduced memory consumption.

Thank you for considering my application, and I look forward to the possibility of working together to advance Generative AI technologies within the OpenVINO framework.

**Comment by adrianboguszewski** [comment]
Thanks for being interested. You will need to fill out our template from https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code and submit it in the GSoC web app. But, of course, we need to wait until the official Google announcement.

@alvoron @dmitry-gorokhov for your visibility

**Comment by alvoron** [comment]
@AtharvaPore01 thank you for submitting your proposal!
we'll bring some details of the task to this thread on the next week.

**Comment by AtharvaPore01** [comment]
Hello @alvoron,

Thank you for acknowledging my proposal submission. I'm grateful for your attention to it.

I eagerly anticipate receiving the task details next week. If there are any specific actions I need to take to advance this project for GSoC 2024, please inform me. I'm enthusiastic about collaborating with both you and @dmitry-gorokhov to kickstart this project.

Looking forward to your response.

@dmitry-gorokhov for your visibility
